src/player-config.json

```diff
@@ -7,14 +7,16 @@
     "preload": "none",
     "autoplay": false,
     "muted": false,
-    "options": {
+    "tweaks": {
```

Comment:
@OrenMe  if we're changing this key we should change also the reference in dash & hls adapters